shared/checks/oval/grub2_default_exists.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="grub2_default_exists" version="1">
+    <metadata>
+      <title>Force IOMMU usage in GRUB2</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Look for argument iommu=force in the kernel line in /etc/default/grub.</description>
+    </metadata>
```

Comment:
This OVAL definition does a different thing than described in title and description.